TheaterSchedule.BLL/Services/ScheduleServices.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TheaterSchedule.DAL;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.BLL
+{
+    public class ScheduleServices : IScheduleServices
+    {
+        private ITheaterScheduleUnitOfWork uow;
+
+        public ScheduleServices(ITheaterScheduleUnitOfWork repo)
+        {
+            uow = repo;
+        }
+
+        public IEnumerable<Schedule> GetListPerformancesByDateRange(DateTime startRange, DateTime endRange)
```

Comment:
rename startRange to startDate (because this variable has only one value, not range)